dashboard/app/views/levels/_form.html.haml

```diff
@@ -11,7 +11,7 @@
       = f.text_field :name, placeholder: 'Level Name'
   / Only enter name to create new records of certain level types.
   - unless @level.new_record? && @level.is_a?(Blockly)
-    = render partial: 'levels/editors/all', locals: {f: f}
+    = render partial: 'levels/editors/all', locals: {f: f} unless @level.is_a? EvaluationQuestion
```

Comment:
None of the common stuff seems necessary for these levels. There's no concepts to relate this to or videos to attach. I don't see the value in putting them on here.